### ISSUE TITLE
Make it clearer when a tensor was quantized symmetrically

### ISSUE THREAD

**Issue by vkuzo** [pytorch/pytorch#51152]
## 🐛 Bug

Symmetric quantization scheme is a special case of an affine quantization scheme.  Currently, the information on whether the tensor was quantized with a symmetric scheme or not is not stored on the quantized Tensor.  This is numerically correct but confusing.

## To Reproduce

1. quantize a tensor (there is no way to specify a qscheme here)
2. verify that printed qscheme on the tensor is affine, even if the intent was to use symmetric
```
>>> import torch                                                               
>>> x = torch.randn(2, 2)                                                      
>>> xq = torch.quantize_per_tensor(x, 0.1, 0, torch.qint8)                     
>>> xq                                                                         
tensor([[-0.3000, -0.4000],                                                    
        [-0.3000,  1.0000]], size=(2, 2), dtype=torch.qint8,                   
       quantization_scheme=torch.per_tensor_affine, scale=0.1, zero_point=0)   
```

At the very least, we should document this behavior.  It would be even better to store qscheme on the tensor.

**Comment by zetyquickly** [comment]
Hello @vkuzo,

Could you please explain what is the indicator that it is a symmetrical affine quantization?

**Comment by supriyar** [comment]
For this, we can update the quantization_scheme in the quantized tensor to only print per-tensor/per-channel information and drop the affine suffix.

**Comment by jerryzh168** [comment]
yeah this is a deliberate choice in the backend so that we don't need to distinguish between symmetric/affine to simplify the kernel implementation, maybe we can document this somewhere
